docs/date-picker/datepicker.md

```diff
@@ -1,8 +1,122 @@
 # DatePicker Components Specification
 
+urik9/datepickerupdate
+* [Description](#description)
 * [Elements](#elements)
+* [Properties](#properties)
+* [Styles](#styles)
 * [Accessibility](#accessibility)
 * [Behavior](#behavior)
+* [Examples](#examples)
+* [Design](#design)
+
+## Description
+A date picker allows users to select dates by using an input or a popup calendar.
+
+## Elements
+* Input Component with Icon Component
+* Calendar View for selecting dates
+
+![Image of basic date picker](./assets/datepicker/DatePickerBasicInput.png)
```

Comment:
this image needs to below the first bullet

Reply:
done